theme/units.ts

```diff
@@ -0,0 +1,14 @@
+interface Units {
```

Comment:
What's the purpose of Units?

Reply:
Are you referring to the file itself or just the interface?

Reply:
Both

Reply:
so the file itself is to contain the base units that will be used in the app. at the moment we have radius and spacing. radius is the standard border-radius that can be used in the app and spacing is the number by which all of the margin and paddings are a product of. 

this sounds a lot like and enum... 😆